contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -31,12 +34,17 @@ contract AutomationRegistry2_3_SetUp is BaseTest {
   address[] internal s_registrars;
 
   IAutomationRegistryMaster2_3 internal registryMaster;
+  ERC20Mock internal link;
+  ERC20Mock internal nonLink;
```

Comment:
nitpick but could we call this something like "USDCoin" or "mockERC20" or something? I think it might make the tests a little easier to read.

Reply:
I like it. will use mockERC20